pkg/skaffold/deploy/labels.go

```diff
@@ -88,7 +89,12 @@ func addLabels(labels map[string]string, accessor metav1.Object) {
 	accessor.SetLabels(kv)
 }
 
-func updateRuntimeObject(client dynamic.Interface, disco discovery.DiscoveryInterface, labels map[string]string, res Artifact) error {
+func addRunIDAnnotation(runID string, accessor metav1.Object) {
+	annotation := map[string]string{constants.RunIDAnnotation: runID}
```

Comment:
Does this keep the existing annotations?

Reply:
good catch, meant to copy the logic for setting the labels but I forgot. updated, added a unit test and updated the helm integration test